crates/sui/tests/protocol_version_tests.rs

```diff
@@ -11,8 +11,8 @@ use test_utils::authority::start_node;
 async fn test_validator_panics_on_unsupported_protocol_version() {
     let dir = tempfile::TempDir::new().unwrap();
     let network_config = sui_config::builder::ConfigBuilder::new(&dir)
-        .with_protocol_version(ProtocolVersion::new(2))
-        .with_supported_protocol_versions(SupportedProtocolVersions::new_for_testing(1, 1))
+        .with_protocol_version(ProtocolVersion::new(3))
```

Comment:
this also will need a revisited test every version. Maybe expected?
@mystenmark

Reply:
This is my bad - you can replace the literals with constants. replace `3` with `ProtocolVersion::MAX + 1` (you are allowed to do `MAX + 1` in the tests), and then the `2` below can become `ProtocolVersion::MAX`.